components/Talk.js

```diff
@@ -32,13 +40,13 @@ const Talk = ({ talk }) => {
 
 Talk.propTypes = {
     talk: PropTypes.shape({
-        name: PropTypes.string.isRequired,
+        name: PropTypes.string,
         date: PropTypes.string,
         speakers: PropTypes.arrayOf(
             PropTypes.shape({
                 name: PropTypes.string.isRequired,
             }).isRequired,
-        ).isRequired,
+        ),
```

Comment:
If you don't pass a `talk` prop, line 8 is gonna crash (Maybe even line 7?)

Reply:
I remove for speakers props. Because "pending" talk have only name date props

Reply:
Oh yeah, my bad